### PR TITLE
assert that chainguard images have attestations for SBOM, vuln, provenance

### PR DESCRIPTION
**Pull request by imjasonh**

### CODE REVIEW

**Review by hectorj2f — APPROVED** [review]
lgtm

**Review by hectorj2f — APPROVED** [review]
lgtm

File: policies/vendors/chainguard/chainguard-images-signed.yaml [hunk 1]
```diff
@@ -20,3 +20,22 @@ spec:
             subject: https://github.com/chainguard-images/images/.github/workflows/release.yaml@refs/heads/main
       ctlog:
         url: https://rekor.sigstore.dev
+      attestations:
```
Comment (vaikas) [inline policies/vendors/chainguard/chainguard-images-signed.yaml:23]: Nit, maybe update the line 10 to reflect more goodness.

**Review by vaikas — COMMENTED** [review]
Thanks!!!

**Review by hectorj2f — APPROVED** [review]

**Review by hectorj2f — APPROVED** [review]

File: policies/templates/attestations/vuln/vuln-maximum-scan-age-rego.yaml [hunk 2]
```diff
@@ -24,7 +24,7 @@ spec:
           subjectRegExp: .*
       attestations:
         - name: vuln-report-scan-max-age
-          predicateType: https://cosign.sigstore.dev/attestation/vuln/v1
+          predicateType: cosign.sigstore.dev/attestation/vuln/v1
```
Comment (hectorj2f) [inline policies/templates/attestations/vuln/vuln-maximum-scan-age-rego.yaml]: this won't work with cosign v2

Comment (imjasonh) [reply policies/templates/attestations/vuln/vuln-maximum-scan-age-rego.yaml]: I'm not sure how to resolve that, and it's unrelated to my change, so I just reverted it.

Comment (hectorj2f) [reply policies/templates/attestations/vuln/vuln-maximum-scan-age-rego.yaml]: We will need to handle both cosign versions. Let me share you an example.

**Pull request merged**